code-of-conduct/index.md

```diff
@@ -4,32 +4,43 @@ title: Code of Conduct
 description: Our Code of Conduct is at the heart of our event.
 ---
 
-All attendees, speakers, sponsors, and volunteers at {{site.title}} and throughout a mentorship are required to agree with the following code of conduct. Organizers will enforce this code throughout {{site.title}} and for the duration of a mentorship. We are expecting cooperation from all participants to help ensure a safe environment for everybody.
+
+All attendees, speakers, sponsors, and volunteers at Ela Conf, throughout a mentorship, on Slack, and community events are required to agree with the following code of conduct. Organizers will enforce this code throughout Ela Conf, the community, and for the duration of a mentorship. We are expecting cooperation from all participants to help ensure a safe environment for everybody.
 
 Ela is a conference and community for adult (18+) marginalized genders.
```

Comment:
Just noticed this has outdated language and we should switch it to "Ela is a conference and community for adult (18+) women (cis and trans), trans men, and genderqueer people.

Reply:
When this merges that _should_ be updated :) Those changes are in master

Reply:
Oh duh 🤦‍♂️  Can you tell this is my first time using this function 😂